model/src/main/java/org/cbioportal/model/Sample.java

```diff
@@ -14,14 +14,31 @@
         BLOOD_NORMAL("Blood Derived Normal"),
         SOLID_NORMAL("Solid Tissues Normal");
 
```

Comment:
We can access it anywhere actually http://stackoverflow.com/a/6280981 . For example I access it in `org.cbioportal.persistence.mybatis.typehandler.SampleTypeTypeHandler`.